posthog/api/action.py

```diff
@@ -440,6 +440,8 @@ def _calculate_people(events: QuerySet):
             people = Person.objects\
                 .filter(team=team, id__in=[p['person_id'] for p in events[0:100]])
 
+            if request.GET.get('breakdown_type') == 'cohort' and request.GET.get('breakdown_value') != 'all users':
+                people = people.filter(Cohort.objects.get(team=team, pk=request.GET['breakdown_value']).people_filter())
```

Comment:
breaking when a specific cohort breakdown is applied and a point is clicked on. debugging showed that`pk=` should be `name`. Is name unique for cohorts model? This looks like it could break if there are multiple cohorts with same name.= Alternatively, the client can be made to actually pass back the pk